README.md

```diff
@@ -0,0 +1,5 @@
+# `errors`
+
+This package provides functionalities close to `errors` (stdlib) & `github.com/pkg/errors` with stacktrace embedding, tagging, etc.
```

Comment:
it should be a drop-in replacement of `errors`  (now that you have join 😄 ) and in terms of function API a drop-in replacement of `pkg/errors`